stable/database/values.yaml

```diff
@@ -328,5 +328,3 @@ openshift:
   enabled: false
   # generally avoid deployment configs, favor deployments, so leave this as false
   enableDeploymentConfigs: false
-  # if openshift is enabled, then also permit routes by default
-  enableRoutes: true
```

Comment:
Do we care about the stuff in incubator still having refs to routes and routes.yaml files present

Reply:
the routes still exist in the incubator. I removed it from here, since there are no routes in the database.